src/main/scala/org/hammerlab/guacamole/likelihood/Likelihood.scala

```diff
@@ -174,6 +178,25 @@ object Likelihood {
       exp(possiblyNormalizedLogLikelihoods)
   }
 
+  def likelihoodsOfGenotypes(elements: Seq[PileupElement],
+                             genotypes: (Genotype, Genotype),
+                             includeAlignment: Boolean,
+                             prior: Genotype => Double,
+                             logSpace: Boolean,
+                             normalize: Boolean): (Double, Double) = {
```

Comment:
Seems like our calling pattern, from outside `Likelihood`, was exclusively either one- or two-genotype cases, and immediately unpacked them with `(0)` and `(1)`, so I thought it good to fold that into the API. I didn't do the one-genotype case just due to laziness / orthogonality to your original change.